### PR TITLE
[FIX] Trigger auto-save when showing TravelModal

### PR DESCRIPTION
**Pull request by brian-man**

### CODE REVIEW

**Comment by elias7905** [comment]
@adamhannigan I was testing for @brian-man and we couldn't seem to get the autosave to work when we clicked on the world button, do you know where we went wrong?

File: src/features/island/hud/components/deliveries/TravelModal.tsx [hunk 1]
```diff
@@ -29,6 +29,7 @@ export const TravelModal: React.FC<Props> = ({ isOpen, onClose }) => {
         centered
         show={isOpen}
         onHide={onClose}
+        OnShow={() => gameService.send("SAVE")}
```
Comment (elias7905) [inline src/features/island/hud/components/deliveries/TravelModal.tsx:32]: It was a capitalisation issue
"OnShow" change to "onShow"

Comment (brian-man) [reply src/features/island/hud/components/deliveries/TravelModal.tsx:32]: Weird.  I quadruple checked that.  Fixing now.

**Review by adamhannigan — APPROVED** [review]
Nice addition 👏 

For context, we have an upcoming task that is dedicated to World Modal UX + Fixes 😄 We will be shipping a lot more improvements

**Pull request merged**